src/android/src/main/java/io/callstack/react/fbads/FBAdsPackage.java

```diff
@@ -31,11 +31,6 @@
     }
 
     @Override
-    public List<Class<? extends JavaScriptModule>> createJSModules() {
```

Comment:
It will fail with compile error in older react native versions if the method is removed. You need to remove the `@override`, but keep the method.